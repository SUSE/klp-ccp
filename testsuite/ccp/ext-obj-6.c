/* Externalizable, but never accessed */
char espr_o[] = "123";

void pu_f(void)
{
	char s[sizeof(espr_o)];
}
