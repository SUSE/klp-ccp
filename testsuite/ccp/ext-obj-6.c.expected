/* klp-ccp: from testsuite/ccp/ext-obj-6.c */
extern char espr_o[4];

void klpp_pu_f(void)
{
	char s[sizeof(espr_o)];
}
